packages/Heading/stories/Heading.stories.mdx

```diff
@@ -0,0 +1,258 @@
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+import { Description } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import Readme from "../README.md";
+import { HeadingStory } from "./Heading.stories.styles";
+import HeadingFocus from "./examples/HeadingFocus";
+import DividerExample from "./examples/DividerExample";
+import UnderlineExample from "./examples/UnderlineExample";
+import ExternalLinkExample from "./examples/ExternalLinkExample";
+import SetWidthExample from "./examples/SetWidthExample";
+import ParagraphExample from "./examples/ParagraphExample";
+import IsLightExample from "./examples/IsLightExample";
+import DisplayLevelExample from "./examples/DisplayLevelExample";
+import Heading from "../src/";
+
+export const params = {
+  options: {
+    isToolshown: true,
+    showPanel: false,
+  },
+};
+
+<Meta title={`${getStoryName("Heading")}/Docs`} component={Heading} parameters={params} />
+
+<Description markdown={Readme} />
+
+## More Examples
+
+### Index
+
+- <a href="#components">Components</a>
+
+  - <a href="#external-link">External Link</a>
+
+- <a href="#heading-levels">Heading Levels</a>
+
+  - <a href="#paragraph">Paragraph</a>
+
+  - <a href="#display-level">Display Level</a>
+
+* <a href="#heading-styles">Heading Styles</a>
+
+  - <a href="#is-light">Is Light</a>
+
+  - <a href="#divider">Divider</a>
+
+  - <a href="#underline">Underline</a>
+
+  - <a href="#set-width">Set Width</a>
+
+### Components
+
+#### External Link
```

Comment:
I think we settled on dropping this example from the docs, and just leaving the "set width" example that _also_ demonstrates this? (But leave it in the Examples).